cl/search/templates/includes/search_result.html

```diff
@@ -32,6 +32,12 @@ <h3 class="bottom serif">
     {% endif %}
   </h3>
 
+  {% if cited_cluster %}
+  <div class="bottom">
+      <span class="meta-data-header">{{ result.citation_depth }} reference{{ result.citation_depth|pluralize }}</span> to {{ cited_cluster.caption|safe|v_wrapper }}
+  </div>
+  {% endif %}
```

Comment:
This is the other spot where we need to guard against `citation_depth` being `None`.

Reply:
I think that this is no longer necessary, since the `citation_depth` variable is now modified with the "default" filter in your patch 23e065c (applied manually by me)?